src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/prisonapi/prisonapidto.kt

```diff
@@ -15,6 +17,26 @@ data class PrisonerAtLocation(
   val assignedLivingUnitId: Long,
 )
 
+data class PrisonerAlert(
+  val alertType: String,
+  val alertCode: String,
+  val active: Boolean,
+  val expired: Boolean,
+) {
+  val isOpenAcct = alertCode == "HA" && active && !expired
+}
+
+data class PrisonerExtraInfo(
+  override val bookingId: Long,
+  override val dateOfBirth: LocalDate,
+  override val receptionDate: LocalDate,
+  val offenderNo: String,
+  val alerts: List<PrisonerAlert> = emptyList(),
+) : PrisonerInfoForNextReviewDate {
+  override val hasAcctOpen = alerts.any(PrisonerAlert::isOpenAcct)
+  override val prisonerNumber = offenderNo
```

Comment:
Prison API's response has a `offenderNo` field instead of calling it `prisonerNumber` like the Offender Search API, so I'm mapping it `offenderNo ` => `offenderNo` to adhere with the `PrisonerInfoForNextReviewDate` interface.